File: android/measure/src/test/java/sh/measure/android/fakes/NoopLogger.kt
package sh.measure.android.fakes

import sh.measure.android.logger.LogLevel
import sh.measure.android.logger.Logger

internal class NoopLogger : Logger {
    override val enabled: Boolean = false

    override fun log(level: LogLevel, message: String, throwable: Throwable?) {
        // No-op
    }
}

File: flutter/packages/measure_flutter/test/utils/noop_logger.dart
import 'package:measure_flutter/src/logger/log_level.dart';
import 'package:measure_flutter/src/logger/logger.dart';

class NoopLogger extends Logger {
  @override
  bool get enabled => true;

  @override
  void log(LogLevel level, String message,
      [Object? error, StackTrace? stackTrace]) {
    // No-op
  }
}
